build-info-extractor-go/src/main/java/org/jfrog/build/extractor/go/extractor/GoZipBallStreamer.java

```diff
@@ -287,4 +323,25 @@ private static String stripFirstPathElement(String path) {
     public void close() throws IOException {
         zipFile.close();
     }
+
+    private String getFirstPathElement(String path) {
```

Comment:
I think all of this function can be replaced by:
```java
StringUtils.substringBefore(path, "/");
```
And a question - Can windows paths with `\` as a separator be here?

Reply:
Nope - if path is "/example/go.mod" -> return value is "/"

Reply:
Got it, and still, those methods looks a bit complicated for me. 
Please consider the following:
```java
private String getFirstPathElement(String path) {
    if (path == null) {
        return "";
    }
    path = StringUtils.removeStart(path, "/");
    return StringUtils.substringBefore(path, "/");
}
```

Reply:
ok NP